test/integration/rtds_integration_test.cc

```diff
@@ -36,7 +38,7 @@ std::string tdsBootstrapConfig(absl::string_view api_type) {
           grpc_services:
             envoy_grpc:
               cluster_name: rtds_cluster
-          set_node_on_first_message_only: true
+          set_node_on_first_message_only: false
```

Comment:
Please revert the test change. We want to default to this true in the next xDS version.

Reply:
Sorry, I think I misread what parts you were commenting on - from what you were saying I thought I might have broken something.

I believe the only node-skipping-related changes I made were here, in the integration testing. The old, to-be-deleted GrpcMuxImpl etc, where it is actually implemented (along with the config loading pipeline to enable it), is unchanged, along with the unit testing. So, no need to be alarmed: this PR does not interfere at all with the node-skipping functionality! It only changes the behavior of the integration tests. I don't think filling-or-not a proto field that is ignored anyways should have any negative effect on the efficacy of the integration testing.

Reply:
OK, thanks for confirming. SG if there is no change to node skipping in the old code.